bin/function_override_wrapper.sh.erb

```diff
@@ -1,6 +1,9 @@
 /usr/bin/env bash -c '
 # load in command and function overrides
-for override_file in <%=function_override_path_binding_for_template%>; do
+system_ls=$(which ls)
+system_grep=$(which grep)
```

Comment:
That could work. I was getting weird issues on Ubuntu with it, however, that will probably need to fixed. It's probably just an update to the regex is all (My whereis grep was returning both the command and the man tarball).